### PR TITLE
LIBWEB-1531 - TikTok Pixel - updating docs to explain 'Use Existing Pixel' setting

### PR DESCRIPTION
**Pull request by joe-ayoub-segment**
Related to SEV - inc-sev2-6364-tiktok-pixel-is-preventing-ajs-from-tracking-events-2023-10-02

Slack: https://twilio.slack.com/archives/C05USP6KDC3/p1696264024954719

JIRA ticket to fix Actions code: https://segment.atlassian.net/browse/LIBWEB-1531

This PR updates the TikTok Pixel docs to provide a clearer explanation of how to use the 'Use Existing Pixel', and the risks involved with disabling it.

### CODE REVIEW

**Comment by joe-ayoub-segment** [comment]
Hi Docs Team, this PR can go out without being dependent on the associated JIRA ticket mentioned in the description above.

File: src/connections/destinations/catalog/actions-tiktok-pixel/index.md [hunk 1]
```diff
@@ -26,11 +26,20 @@ This destination is maintained by TikTok. For any issues with the destination, [
...
 7. Toggle on the Destination using the **Enable Destination** toggle.
 8. Click **Save Change**.
 
+<a id="use-existing-pixel-setting"></a>
+## The "Use Existing Pixel" setting
+
+The TikTok Pixel Destination's default behavior is to load a TikTok Pixel Javascript library onto your website when the page loads. However if you already have TikTok Pixel on your website before enabling this Destination, you might prefer that Segment detects and uses your pre-existing TikTok Pixel Javascript library instead of loading a new TikTok Pixel library. 
+
+Toggling "off" the "Use Existing Pixel" will prevent Segment from loading a TikTok Pixel library onto the web page. Instead, Segment will attempt to detect your pre-loaded TikTok Pixel library and will use it when forwarding events to TikTok.
```
Comment (chrisradek) [inline src/connections/destinations/catalog/actions-tiktok-pixel/index.md:39]:
```suggestion
Toggling "on" the "Use Existing Pixel" will prevent Segment from loading a TikTok Pixel library onto the web page. Instead, Segment will attempt to detect your pre-loaded TikTok Pixel library and will use it when forwarding events to TikTok. 
```

File: src/connections/destinations/catalog/actions-tiktok-pixel/index.md [hunk 2]
```diff
@@ -26,11 +26,20 @@ This destination is maintained by TikTok. For any issues with the destination, [
...
 8. Click **Save Change**.
 
+<a id="use-existing-pixel-setting"></a>
+## The "Use Existing Pixel" setting
+
+The TikTok Pixel Destination's default behavior is to load a TikTok Pixel Javascript library onto your website when the page loads. However if you already have TikTok Pixel on your website before enabling this Destination, you might prefer that Segment detects and uses your pre-existing TikTok Pixel Javascript library instead of loading a new TikTok Pixel library. 
+
+Toggling "off" the "Use Existing Pixel" will prevent Segment from loading a TikTok Pixel library onto the web page. Instead, Segment will attempt to detect your pre-loaded TikTok Pixel library and will use it when forwarding events to TikTok. 
+
+Care should be taken when toggling the "Use Existing Pixel" setting to "off" as doing so could result in disruption to data collection if Segment doesn't detect TikTok Pixel on the website.
```
Comment (chrisradek) [inline src/connections/destinations/catalog/actions-tiktok-pixel/index.md:41]:
```suggestion
Care should be taken when toggling the "Use Existing Pixel" setting to "on" as doing so could result in disruption to data collection if Segment doesn't detect TikTok Pixel on the website. 
```

**Review by chrisradek — CHANGES_REQUESTED** [review]
I think you flipped "off" vs "on"

**Review by chrisradek — APPROVED** [review]

**Review by pwseg — APPROVED** [review]

**Pull request merged**